--- lib/Foswiki/Plugins/NatSkinPlugin/HtmlTitle.pm.old
+++ lib/Foswiki/Plugins/NatSkinPlugin/HtmlTitle.pm
@@ -68,7 +68,7 @@
   if (Foswiki::Func::getContext()->{DBCachePluginEnabled}) {
     #print STDERR "using DBCachePlugin\n";
     require Foswiki::Plugins::DBCachePlugin;
-    return Foswiki::Plugins::DBCachePlugin::getTopicTitle($web, $topic);
+    return Foswiki::Func::getTopicTitle($web, $topic);
   }
 
   #print STDERR "using foswiki core means\n";

